.github/workflows/publish.yml

```diff
@@ -3,24 +3,28 @@ name: Publish
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
-      - 'v[0-9]+.[0-9]+.[0-9]+-preview.[0-9]+'
-      - 'v[0-9]+.[0-9]+.[0-9]+-beta.[0-9]+'
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-preview.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-beta.[0-9]+"
 
 jobs:
   publish:
+    name: Publish
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'playcanvas'
+    if: ${{ github.repository_owner == 'playcanvas' }}
+    permissions:
+      contents: read
+      id-token: write
     steps:
       - name: Check out code
         uses: actions/checkout@v6
 
-      - name: Set up Node.js 22.x
+      - name: Set up Node.js 24.x
```

Comment:
I would just remove the version here. Renovate never updates this when auto-updating `node-version`.